tests/main/interfaces-hotplug-basic/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Basic test to ensure udev monitor doesn't error out and is enabled.
+
+execute: |
+    # shellcheck source=tests/lib/journalctl.sh
+    . "$TESTSLIB/journalctl.sh"
+
+    # Restart so that we capture early startup messages in the logs
+    systemctl restart snapd.{socket,service}
+
+    # Feature flag doesn't need to be set for udev monitor to start, but make sure
+    # the flag is accepted (TODO: add some checks based on the flag on/off once
+    # hotplug works).
+    echo "Ensure feature flag is enabled"
+    snap set core experimental.hotplug=true
+
+    if [ "$(snap get core experimental.hotplug)" != "true" ]; then
+        echo "Expected experimental.hotplug feature to be enabled"
+        exit 1
+    fi
+
+    echo "Ensure that udev monitoring is enabled and doesn't fail"
+    if get_journalctl_log | MATCH "Failed to start udev monitor:"; then
```

Comment:
If you tweak the error messages please ensure that those are also reflected here (e.g. `cannot start udev monitor`)

Reply:
Yes, this PR will be updated once the base PR lands.

Reply:
This sort of test is extremely unreliable, and @zyga hints at the issue: any test that verifies that a log message is _not_ observed is bound to become irrelevant once the code progresses.

In other words, the test should check that the feature indeed works, instead of observing a log message that claims that it doesn't work. The log message is irrelevant anyway. It may be working or not independent from whether the message was logged.